hello_world/src/hello-world.cpp

```diff
@@ -1,3 +1,3 @@
-#include "hello-world.hpp"
+#include "../include/hello-world.hpp"
```

Comment:
Don't use includes relative the source file; includes should be relative to the include path. It should just be `#include "hello-world.hpp"`